sample/kotlin/src/main/kotlin/com/datadog/android/sample/gdpr/GdprDialogFragment.kt

```diff
@@ -47,6 +49,12 @@ internal class GdprDialogFragment : DialogFragment() {
         return rootView
     }
 
+    @OptIn(ExperimentalRumApi::class)
+    override fun onResume() {
+        super.onResume()
+        GlobalRumMonitor.get().addViewLoadingTime()
```

Comment:
same: I guess we can have multiple calls for the same view?

Reply:
same as above